validator/remote/src/integration-test/java/tech/pegasys/teku/validator/remote/typedef/OkHttpValidatorTypeDefClientTest.java

```diff
@@ -323,6 +341,71 @@ void blockV3ShouldFallbacksToBlockV2WhenNotFound()
     assertThat(secondRequest.getPath()).startsWith("/eth/v1/validator/blinded_blocks");
   }
 
+  @TestTemplate
+  void postValidators_MakesExpectedRequest() throws Exception {
+    mockWebServer.enqueue(new MockResponse().setResponseCode(SC_NO_CONTENT));
+
+    okHttpValidatorTypeDefClient.postStateValidators(List.of("1", "0x1234"));
+
+    final RecordedRequest request = mockWebServer.takeRequest();
+    assertThat(request.getMethod()).isEqualTo("POST");
+
+    assertThat(request.getPath()).contains(ValidatorApiMethod.GET_VALIDATORS.getPath(emptyMap()));
+    assertThat(request.getBody().readUtf8()).isEqualTo("{\"ids\":[\"1\",\"0x1234\"]}");
+  }
+
+  @TestTemplate
+  public void postValidators_WhenNoContent_ReturnsEmpty() {
+    mockWebServer.enqueue(new MockResponse().setResponseCode(SC_NO_CONTENT));
+
+    assertThat(okHttpValidatorTypeDefClient.postStateValidators(List.of("1"))).isEmpty();
+  }
+
+  @ParameterizedTest
+  @ValueSource(ints = {SC_BAD_REQUEST, SC_NOT_FOUND, SC_METHOD_NOT_ALLOWED})
+  public void postValidators_WhenNotExisting_ThrowsException(final int responseCode) {
+    mockWebServer.enqueue(new MockResponse().setResponseCode(responseCode));
+
+    assertThatThrownBy(() -> okHttpValidatorTypeDefClient.postStateValidators(List.of("1")))
+        .isInstanceOf(PostStateValidatorsNotExistingException.class);
+  }
```

Comment:
just re-adding my comment here in case we miss it
basically this test being mixed in with `@TestTemplate` tests is causing weird errors, and it should just be separated out into a different suite so that its not mixed with tests being run with `@TestTemplate`